scripts/HUMAnN2_pipeline.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#$ -o ./log/
+#$ -e ./log/
+
+
+#STEPS
+# run the humann2 
+# renormalize to copies per million and relative abundance 
+# map to KO and EC terms
+# remove the intermediate files
+
+#specify paths 
```

Comment:
should the user be chaning anything else besides those paths below (lines 14-17)? if not, please mark this section clearly, so that there is no doubt the user should only change those paths. If there is anything else to be changed, I am a big fan of grouping those things together, so that the user does not need to go through the whole script to change variables.

Reply:
here user only needs to change the paths only. Will automate the workflow and make some of the things clear.